### DISCUSSION TITLE
Focus tab on top of the scrolled list

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#74] [Ideas]
If auto restore is enabled, it focus the tab when clicked in the all tabs view which is good. But the problem is, that when clicked, the selected tab appear sometimes to near at the bottom or in the middle and not on on top of the scrolled view. I don't mean that the tab has to move on top of the tab list when clicked. Only the tab view should scroll to the clicked tab as if it were the first in the scrolled list. This would be good for auto restored tabs like the header tabs, but also for other tabs. I would be glad about such an addition.

**Comment by madprops** [comment]
I'm not gonna make it scroll at the top, but i'll change it to `nearest`, as it should be.

**Comment by N3C2L** [comment]
Thanks!

**Comment by N3C2L** [comment]
This idea has been implemented, so I can close this thread.
